helios/always-succeeds.js

```diff
@@ -0,0 +1,5 @@
+spending always_succeeds
+
+func main(_, _, _) -> Bool {
```

Comment:
everything is ommited.
So it doesn't make sense if I have Datum but no Redeemer;
Can I have Redeemer without Datum?
Can I have Redeemer without context? :-1: